build.gradle

```diff
@@ -1,17 +1,23 @@
 ext {
     GROUPID = 'de.golfgl.gdxgamesvcs'
     VERSION = '1.1.0'
-
 }
 
 buildscript {
     repositories {
-        maven { url 'https://plugins.gradle.org/m2/' }
+        mavenLocal()
+        mavenCentral()
         google()
+        maven { url "https://plugins.gradle.org/m2/" }
+        maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
+        maven { url 'https://developer.huawei.com/repo/' }
+        gradlePluginPortal()
     }
     dependencies {
         classpath 'org.wisepersist:gwt-gradle-plugin:1.0.6'
-        classpath 'com.android.tools.build:gradle:3.2.0'
+        classpath 'com.android.tools.build:gradle:4.2.2'
+        classpath 'com.google.gms:google-services:4.3.10'
```

Comment:
I think it is better to leave dependencies that are only used for single modules in these modules. It is better documented where it is used this way.

Reply:
I will look into this and move the dependencies inside the modules they are used.

Reply:
I have moved the google services dependencies to android-gpgs submodule and huawei related stuff to android-huawei submodule. Should be ok now.